docs/source/app-dev/app-arch.rst

```diff
@@ -136,23 +134,13 @@ The interaction of a Daml application with the ledger is inherently asynchronous
 There are several things that can fail during this time window: the application can crash, the participant node can crash, messages can be lost on the network, or the ledger may be just slow to respond due to a high load.
```

Comment:
For readability purposes this might work better as an unordered list than a long sentence.

Reply:
Done.